ocpp/v201/call_result.py

```diff
@@ -323,62 +327,62 @@ class SecurityEventNotification:
 
 @dataclass
 class SendLocalList:
-    status: str
-    status_info: Optional[Dict] = None
+    status: enums.SendLocalListStatusEnumType
+    status_info: Optional[datatypes.StatusInfoType] = None
     custom_data: Optional[Dict[str, Any]] = None
 
 
 @dataclass
 class SetChargingProfile:
-    status: str
-    status_info: Optional[Dict] = None
+    status: enums.ChargingProfileStatusEnumType
+    status_info: Optional[datatypes.StatusInfoType] = None
     custom_data: Optional[Dict[str, Any]] = None
 
 
 @dataclass
 class SetDisplayMessage:
-    status: str
-    status_info: Optional[Dict] = None
+    status: enums.DisplayMessageStatusEnumType
+    status_info: Optional[datatypes.StatusInfoType] = None
     custom_data: Optional[Dict[str, Any]] = None
 
 
 @dataclass
 class SetMonitoringBase:
-    status: str
-    status_info: Optional[Dict] = None
+    status: enums.GenericStatusEnumType
```

Comment:
<img width="841" alt="image" src="https://github.com/user-attachments/assets/dfe7176b-97ab-482d-93d7-7213e6f508fc" />

Should we use `GenericDeviceModelStatusEnumType` or `GenericStatusEnumType`?

Reply:
changed to `GenericDeviceModelStatusEnumType`